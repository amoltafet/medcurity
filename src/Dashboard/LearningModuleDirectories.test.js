import renderer from 'react-test-renderer';
import React from 'react';
import LearningModulesDirectories from './LearningModuleDirectories';

test('renders LoginPage', () => {
  const tree = renderer.create(<LearningModulesDirectories />).toJSON();
  expect(tree).toMatchSnapshot();
});