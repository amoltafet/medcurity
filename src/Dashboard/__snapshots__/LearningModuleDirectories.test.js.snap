// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LoginPage 1`] = `
Array [
  <div
    className="LearningModulesDirectoriesDashboard ms-2 row"
  />,
  <div
    className="d-grid gap-2"
  />,
]
`;
